coda/coda_mdstore/urls.py

```diff
@@ -31,42 +32,9 @@
     url(r'^about/$', views.about, name='about'),
     url(r'^robots.txt$', views.shooRobot, name='robots'),
     url(r'^feed/$', views.AtomSiteNewsFeed(), name='feed'),
+    url(r'^resourceindex\.xml$', sitemap_views.index),
+    url(r'^resourcelist-(?P<section>.+)\.xml$', sitemap_views.sitemap, name='sitemaps_views'),
+    url(r'changelist\.xml$', changelist),
```

Comment:
resourceindex.xml, resourcelist-001.xml, and changelist.xml are all erroring for me with a `TypeError` about arguments. Maybe you need to pass `sitemaps` as an argument as it looks like was done before? If these URLs are also erroring for you and the tests didn't catch it, we should probably add tests that will show a failure before fixing.

Reply:
It seems like they are giving `TypeError` for me as well. And I didn't realize about it 'cos they weren't caught by the tests. I think adding new tests to check these is a good idea.